app/events/donation_destroy_event.rb

```diff
@@ -0,0 +1,15 @@
+class DonationDestroyEvent < Event
+
+  # @param donation [Donation]
+  def self.publish(donation)
+    self.create(
+      eventable: donation,
+      organization_id: donation.organization_id,
+      event_time: Time.zone.now,
+      data: EventTypes::InventoryPayload.new(
+        items: EventTypes::EventLineItem.from_line_items(donation.line_items, from: donation.storage_location_id)
+      )
```

Comment:
This is one that doesn't ahve the `.as_json`

Reply:
Huh. Wondering if `as_json` is even needed if tests are passing with this :thinking: